drizzlepac/hlautils/analyze.py

```diff
@@ -84,7 +83,7 @@ def analyze_data(inputFileList, **kwargs):
 
```

Comment:
Variable should not be camelcase. Rename 'acsFiltNameList' to 'acs_filt_name_list' and find/replace the remaining instances.